packages/stryker-mocha-runner/test/unit/MochaTestRunnerSpec.ts

```diff
@@ -1,7 +1,7 @@
 import * as path from 'path';
 import { EventEmitter } from 'events';
 import * as Mocha from 'mocha';
-import * as log4js from 'log4js';
+import * as log4js from 'stryker-api/logging';
```

Comment:
we call it log4js here and import the entire module

Reply:
Refactored the name, but the entire module is needed in order to mock away the `getLogger` method.